crates/zed/src/languages/hcl/config.toml

```diff
@@ -0,0 +1,13 @@
+name = "HCL"
+path_suffixes = ["hcl", "hcl2"]
+line_comments = ["# "]
```

Comment:
Hm, good point. I think I had `//` in there previously but backed it out for some reason. It's valid syntax so we should highlight it accordingly.

Seems the first element of this array is used by the cmd-/ when entering comments, so I think having it default to `#` makes sense even if it's different to vscode. 

Both Packer and Nomad default to `#` for comments in all the HCL examples. For Terraform, the [Syntax Guide](https://developer.hashicorp.com/terraform/language/syntax/configuration#comments) states

> The `#` single-line comment style is the default comment style and should be used in most cases. Automatic configuration formatting tools may automatically transform `//` comments into `#` comments, since the double-slash style is not idiomatic.

I think I'm okay with people opening code previously edited and being able to insert `//` by hand to continue their own convention, but using "editor: toggle comments" inserts `#` by default.